src/labels/CustomLabels.labels

```diff
@@ -8679,6 +8687,30 @@ To check your existing settings, go to Duplicate Rules under Setup. For any rule
         <shortDescription>stgHelpUseDatedConvRates</shortDescription>
         <value>When checked, Dated Exchange Rates are used for currency conversion. If unchecked, the standard exchange rate is used even if Advanced Currency Management is enabled.</value>
     </labels>
+    <labels>
+        <fullName>stgInstallmentOppDisableAll</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>stgInstallmentOppDisableAll</shortDescription>
```

Comment:
The shortDescriptions on these labels should explain what the labels are used for as opposed to be identical to the label fullName.